pylint/__pkginfo__.py

```diff
@@ -40,7 +40,7 @@
     version += "-dev" + str(dev_version)
 
 install_requires = [
-    "astroid==2.5.0",
+    "astroid>=2.5.1,<2.7",
```

Comment:
I would leave it at that and don't change it for the `pylint` release. What do you think?

Reply:
Yeah I think we already discussed this before, but putting <2.7 here means we need to think about retro-compatibility with pylint 2.7.2 until astroid 2.6+ ? Shouldn't it be <2.6 ?

Reply:
I'll merge and prepare the release of 2.7.2, I can change this later following this discussion without re-running github actions for no reason.

Reply:
Yeah we did, but it was a while back and we didn't resolve it. That's why I'm bringing it back up.

Until a moment ago I would have argued that `<2.7` does work, since once `2.6` is released a new pylint version will sure follow. But assuming this release gets into debian it might actually be better to limit it to `<2.6`.

Reply:
We just need to update it to `<2.7` on master once the release is done.